WeScanTests/RectangleFeaturesFunnelTests.swift

```diff
@@ -92,17 +92,19 @@ final class RectangleFeaturesFunnelTests: XCTestCase {
         
         let expectation = XCTestExpectation(description: "Funnel add callback")
         expectation.isInverted = true
-        
-        for i in 0 ..< count {
-            let rectangleFeature = i % 2 == 0 ? type1RectangleFeatures[i] : type2RectangleFeatures[i]
-            funnel.add(rectangleFeature, currentlyDisplayedRectangle: currentlyDisplayedRect, completion: { (rectFeature) in
-                
-                currentlyDisplayedRect = rectFeature
-                if i >= funnel.maxNumberOfRectangles {
-                    expectation.fulfill()
-                }
-            })
-        }
+      
+      // TODO: Fix this
```

Comment:
This should be fixed before merging in apparently.

Reply:
I've fix this will submit another PR to remove this TODO.